content/news/2003-01-18-installfest.md

```diff
@@ -0,0 +1,17 @@
+Title: InstallFest
+Tags: installfest
+Event: 2003-01-18 10:30 am to 4:00 pm
+Speaker: None
+Location: Sonoma State Campus Schulz Info Center, Room 1121 (see below for specifics)
+Author: Eric Eisenhart
+
+WHAT: NBLUG InstallFest
```

Comment:
I didn't know `<br>` tags were legal in Markdown - I'll make the appropriate changes right away.